docs/verifier.rst

```diff
@@ -0,0 +1,172 @@
+**manticore-verifier** needs to be pointed to a the target contract containing any number of property methods.
+The target contract is the entry point of the exploration. It needs to initilize any internal structure or external contracts to a correct initial state. All methods of this contract matching the property name criteria will be tested. ::
+
+   --contract_name CONTRACT_NAME The target contract name defined in the source code
+
+
+User accounts
+=============
+You can specify what are the accounts used in the exploration.
+Normaly you do not want the owner or deployer of the contract to send the symbolic transaction and to use as eparated unused account to actually check the property methods.
+There are 3 types of user accounts:
+
+    - deployer:  The account used to create the target contract
+    - senders: A set of accounts used to send symbolic transactions. Think that  these transactions are the ones trying to put the contract in a state that makes the property fail.
+    - psender: The account used as caller to test tha actual property methos
+
+
+You can specify those via commanline arguments ::
+
+    --deployer DEPLOYER   (optional) address of account used to deploy the contract
+    --senders SENDERS     (optional) a comma separated list of sender addresses.
+                           The properties are going to be tested sending
+                           transactions from these addresses.
+    --psender PSENDER     (optional) address from where the property is tested
+
+
+Or, if you prefer, you can specify a yaml file like this ::
+
+    deployer: "0x41414141414141414141" 
+    sender: ["0x51515151515151515151", "0x52525252525252525252"] 
+    psender: "0x616161616161616161"
+
+If you specify the accounts both ways the commandline takes precedence over the yaml file.
+If you do not provide specific accounts **manticore-verifier** will choose them for you.
+
+
+Stopping condition
+==================
+The exploration will continue to sent symbolic transactions until one of the stopping criteria is met.
+
+Maximun number of trasactions
+-----------------------------
+You can be interested only in what could happen under a number of transactions. After a maximun number of transactions is reached the explorations ends. Properties that had not be found to be breakable are considered a pass.
+You can modify the max number of transactions to test vis a command line argument, otherwise it will stop at 3 transactions. ::
+
+     --maxt MAXT           Max transaction count to explore
+ 
+Maximun coverage % attained
+---------------------------
+By default if a transaction does not produce new coverage the exploration is stopped. But you can add a further constraining,  if the provided coverage percentage is obtained stop. Not that this is the total % of runtime bytecode covered. By default compilers add dead code and also in this case the rntime contains the code of the propertie methods. So use with care. ::
```

Comment:
```suggestion
By default, if a transaction does not produce new coverage, the exploration is stopped. But you can add a further constraint so that if the provided coverage percentage is obtained, stop. Note that this is the total % of runtime bytecode covered. By default, compilers add dead code, and also in this case the runtime contains the code of the properties methods. So use with care. ::
```